cgm/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+autoreconf -fi
+./configure --prefix=$PREFIX --enable-optimize --disable-debug --enable-shared
```

Comment:
Yes - probably best to leave it out for now, and we can work on adding CGM when OCE is ready for primetime - hopefully soon(TM).